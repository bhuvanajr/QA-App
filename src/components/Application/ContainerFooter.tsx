import React from 'react';

export const ContainerFooter: React.FC = () => {
  return (
    <div className="container">
      <div className="row ">
        <div className="col-sm-auto">
          <ul className="nav flex-column flex-sm-row">
            <li className="nav-item">
              <a className="nav-link" href="#" target="_blank" rel="noreferrer">
                Privacy
              </a>
            </li>
            <li className="nav-item">
              <a className="nav-link" href="#" target="_blank" rel="noreferrer">
                Cookies
              </a>
            </li>
            <li className="nav-item">
              <a className="nav-link" href="https://www.energiebespaarders.nl/" target="_blank" rel="noreferrer">
                <div data-label="footer.copyright">Â© DEB</div>
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  );
};
