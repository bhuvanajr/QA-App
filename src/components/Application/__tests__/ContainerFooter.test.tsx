import { render } from '@testing-library/react';
import React from 'react';
import { ContainerFooter } from '../ContainerFooter';

test('should render ContainerFooter correctly', () => {
  const { container } = render(<ContainerFooter />);
  expect(container).toMatchSnapshot();
});
