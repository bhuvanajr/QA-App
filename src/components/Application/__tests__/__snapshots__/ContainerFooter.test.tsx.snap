// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ContainerFooter correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="row "
    >
      <div
        class="col-sm-auto"
      >
        <ul
          class="nav flex-column flex-sm-row"
        >
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="#"
              rel="noreferrer"
              target="_blank"
            >
              Privacy
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="#"
              rel="noreferrer"
              target="_blank"
            >
              Cookies
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="#"
              rel="noreferrer"
              target="_blank"
            >
              <div
                data-label="footer.copyright"
              >
                Â© QA Tool
              </div>
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
